### PR TITLE
Neccessary changes to update Motor and move to Python 3.7

### PR DESCRIPTION
**Pull request by ikalnytskyi**
And also make CI build green (green, not great) again.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +1,12 @@
...
-      env: TOXENV=py36
-    - python: pypy3
-      env: TOXENV=pypy
-    - python: 3.6
+    - python: 3.7
+      env: TOXENV=py37
+    - python: 3.7
       env: TOXENV=openapi
-    - python: 3.6
+    - python: 3.7
```
Comment (malor) [inline .travis.yml]: Right about time! 3.8 is coming soon :)

File: xsnippet/api/database.py [hunk 2]
```diff
@@ -28,19 +28,7 @@ def create_connection(conf):
     mongo = AsyncIOMotorClient(conf['DATABASE_CONNECTION_URI'])
 
     # get_default_database returns a database from the connection string
```
Comment (malor) [inline xsnippet/api/database.py]: nit: s/get_default_database/get_database

File: xsnippet/api/database.py [hunk 3]
```diff
@@ -28,19 +28,7 @@ def create_connection(conf):
...
-    db = mongo.get_default_database()
-
-    # ID incrementer is used to auto increment record ID if nothing
-    # is explicitly passed.
-    #
-    # ID processor is used for auto converting domain IDs to and from
-    # database ones (i.e. 'id' -> '_id' and vice versa).
-    #
-    # SON manipulators are applied in reversed order.
-    db.add_son_manipulator(_IdIncrementer())
```
Comment (malor) [inline xsnippet/api/database.py]: They have finally removed SON manipulators, hah. `_id` trickery  is a bit annoying, but we can revisit this later.

Comment (ikalnytskyi) [reply xsnippet/api/database.py]: > They have finally removed SON manipulators

Yeah. Now I think it was a bad idea. Even though it was mine :P

**Pull request merged**